frontend/app/routes/$lang+/_public+/apply+/$id+/tax-filing.tsx

```diff
@@ -65,7 +71,7 @@ export async function action({ request, params }: ActionFunctionArgs) {
     state: { taxFiling2023: parsedDataResult.data },
   });
 
-  return parsedDataResult.data.taxFiling2023 === 'FALSE' ? redirectWithLocale(request, `/apply/${id}/file-your-taxes`, sessionResponseInit) : redirectWithLocale(request, `/apply/${id}/date-of-birth`, sessionResponseInit);
+  return parsedDataResult.data === 'FALSE' ? redirectWithLocale(request, `/apply/${id}/file-your-taxes`, sessionResponseInit) : redirectWithLocale(request, `/apply/${id}/date-of-birth`, sessionResponseInit);
```

Comment:
**Nitpick:** Would this be more readable?

```suggestion
  const redirectUrl = parsedDataResult.data === 'FALSE' ? `/apply/${id}/file-your-taxes` : `/apply/${id}/date-of-birth`;
  return redirectWithLocale(request, redirectUrl, sessionResponseInit);
```

It helps distinguish the difference in the result better in my opinion. Feel free to resolve if you disagree.

Reply:
I've made the same comment but different to follow other pages convention for now.

https://github.com/DTS-STN/canadian-dental-care-plan/pull/744#discussion_r1532752196